.github/workflows/packaging.yml

```diff
@@ -302,8 +302,14 @@ jobs:
       - name: Extract source archive
         run: |
           tar xf apache-arrow-adbc-${{ steps.info.outputs.VERSION }}.tar.gz
+
+          source ./apache-arrow-adbc-${{ steps.info.outputs.VERSION }}/dev/release/versions.env
+
           mv apache-arrow-adbc-${{ steps.info.outputs.VERSION }} adbc
-          mv apache-arrow-adbc-${{ steps.info.outputs.VERSION }}.tar.gz adbc/ci/linux-packages/
```

Comment:
It's not related to this PR but we can remove `echo "VERSION=${VERSION}" >> "$GITHUB_OUTPUT"` and always use `${VERSION}` instead of `${{ steps.info.outputs.VERSION }}`.

Reply:
ok, let's see if I did that right...